### DISCUSSION TITLE
Crossed seven

### DISCUSSION THREAD

**Discussion by BrianTuffin** [psb1558/Junicode-font#90] [Q&A] [answered]
I have a preference for crossed sevens. Is there any chance of adding one to both the standard and oldstyle numerals in Junicode?

**Comment by psb1558** [comment, marked as answer]
When this query has [come up](https://github.com/adobe-fonts/source-sans/issues/93) [elsewhere](https://www.quora.com/Why-do-so-few-typefaces-have-a-line-through-the-7-numeral), the answer has been that the cross on a seven is a feature of handwriting (where one and seven are often hard to distinguish) and rarely found in fonts, except in a few specialty fonts. There's also the problem of how to provide access to the feature. OpenType has a "zero" feature tag for the crossed zero, but no feature tag for a crossed seven. Junicode's cvNN and ssNN features are just about full up.

So, regretfully, the answer is no.

**Reply by BrianTuffin** [reply]
Thank you... it was just a whim.

**Reply by jsbien** [reply]
The question is how do you want to access them. Perhaps PUA is an acceptable solution.
